hera_mc/cm_active.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, session=None, at_date='now'):
         self.info = None
         self.apriori = None
         self.geo = None
+        self.pytest_param = False
```

Comment:
Putting test-only activated code like this in the real code feels a bit clunky. Is this really the best way?

Reply:
I'm not sure I'd go with "best", but maybe "reasonable" in this instance.  I originally passed a variable, but I actually thought that was worse in this instance.

Reply:
I guess I'm struggling to understand the purpose of this, maybe just add a detailed comment about what it achieves that you can't achieve otherwise.